docs/tutorial/force_calibration.rst

```diff
@@ -212,3 +212,102 @@ If we compare the different fits, we can see that the Lorentzian model with diod
 While this is true for this particular dataset, no general statement can be made about the bias of fitting a Lorentzian rather than the hydrodynamically correct power spectrum.
 If low bias is desired, one should use the hydrodynamically correct model.
 On regular sensors, it is best to fit the hydrodynamically correct model with the diode model enabled.
+
+Active Calibration
+------------------
+
+For certain applications, passive force calibration, as described above, is not sufficiently accurate.
+Using active calibration, the accuracy of the calibration can be improved, because active calibration uses fewer assumptions than passive calibration.
+
+When performing passive calibration, we base our calculations on a theoretical drag coefficient.
+This theoretical drag coefficient depends on parameters that are only known with limited precision: the diameter of the bead and the viscosity.
+This viscosity in turn depends strongly on the local temperature around the bead, which is typically poorly known.
+
+During active calibration, the trap or nanostage is oscillated sinusoidally.
+These oscillations result in a driving peak in the force spectrum.
+Using power spectral analysis, the force can then be calibrated without prior knowledge of the drag coefficient.
+
+When the power spectrum is computed from an integer number of oscillations, the driving peak is visible at a single data point at :math:`f_{drive}`.
+
+.. image:: driving_input.png
+
+The physical spectrum is then given by a thermal part (like before) and an active part:
```

Comment:
this is the first time in the docs that we refer to the passive Lorentzian as "thermal". It might be good to mention this term up above when talking about the thermal/passive fit just for continuity and context

Reply:
Good point. I've added `Passive calibration is also often referred to as thermal calibration.` to the section above.